src/os/bluestore/BitmapFreelistManager.cc

```diff
@@ -82,7 +82,7 @@ int BitmapFreelistManager::create(uint64_t new_size, uint64_t granularity,
     // set past-eof blocks as allocated
     _xor(size, blocks * bytes_per_block - size, txn);
   }
-  dout(10) << __func__
+  dout(0) << __func__
```

Comment:
I think 10 was better.

Reply:
The primary intent is to be in sync with shutdown/init methods which dout at 1.
Just to avoid shutdown as the first output from freelist on mkfs.
But dout(0) is rather too low, I agree. Increasing to 1...